lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs

```diff
@@ -146,6 +146,7 @@ internal sealed class REEFIMRUClient : IIMRUClient
                 .AddDriverConfiguration(imruDriverConfiguration)
                 .AddGlobalAssemblyForType(typeof(IMRUDriver<TMapInput, TMapOutput, TResult>))
                 .SetJobIdentifier(jobDefinition.JobName)
+                .SetDriverMemory(5000)
```

Comment:
Right, it is required by IMRU client project. Just don't want to have a PR for this line only. And the change in PR and this line are required by IMRU client project.